addons/l10n_ph/wizard/generate_2307_wizard.py

```diff
@@ -74,21 +24,12 @@ def action_generate(self):
         """
         self.ensure_one()
 
-        file_data = io.BytesIO()
-        workbook = xlwt.Workbook(encoding='utf-8')
-        worksheet = workbook.add_sheet('Form2307')
-
-        for index, col_header in enumerate(COLUMN_HEADER_MAP.keys()):
-            worksheet.write(0, index, label=col_header)
-
-        self._write_rows(worksheet, self.moves_to_export)
-
-        workbook.save(file_data)
-        file_data.seek(0)
-        self.generate_xls_file = base64.b64encode(file_data.read())
+        bir_exporter = Bir2307Exporter('Form2307', self.moves_to_export, file_format='xls')
```

Comment:
Will you ever need to use this class differently than just doing
- Create exporter
- Export
- Close exporter
?

If not, you could simply create a method in util that does all that at once and returns the file right away in one line.
self.xls_file = utils._export_bit_2307('Form2307', self.moves_to_export, file_format='xls')

This would make it a bit easier to use